<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerFilterDialog</class>
 <widget class="QDialog" name="playerFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>180</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>201</width>
     <height>145</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="expLabel">
      <property name="text">
       <string>经验值</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="levelCntLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="gradeLabel">
      <property name="text">
       <string>等级</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="levelCntLabel">
      <property name="text">
       <string>已闯关卡数</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="expLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nicknameLabel">
      <property name="text">
       <string>昵称</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nicknameLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="gradeLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
